### PR TITLE
RIA-7823 Create UpdateHearingRequest flow

### PR DESCRIPTION
**Pull request by MiguelMartinRojas**
### Jira link (if applicable)

RIA-7823 Create UpdateHearingRequest flow

### Change description ###
RIA-7823 Create UpdateHearingRequest flow

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HearingsUpdateHearingRequest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        return asylumCaseCallbackApiDelegator.delegate(callback, hearingsApiEndpoint + midEventPath);
+    }
+
+    private static void setHearingLocationDetails(AsylumCase asylumCase) {
+        Optional<String> hearingLocation = asylumCase.read(HEARING_LOCATION_VALUE);
+        if (hearingLocation.isPresent()) {
+            String hearingCenterValue = HearingCentre.getValueByEpimsId(hearingLocation.get());
+            asylumCase.write(HEARING_LOCATION_VALUE, hearingCenterValue);
+            DynamicList locationsList = new DynamicList(
+                    new uk.gov.hmcts.reform.iacaseapi.domain.entities.Value(hearingLocation.get(), hearingCenterValue),
```
Comment (robin-ieong) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HearingsUpdateHearingRequest.java:95]: Can we import the class?

Comment (MiguelMartinRojas) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HearingsUpdateHearingRequest.java:95]: I can't it collides with the Value spring annotation https://github.com/hmcts/ia-case-api/blob/e44cbd36298166463ddfe3312f1ec2e9eef3657b/src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HearingsUpdateHearingRequest.java#L37-L37

**Review by robin-ieong — APPROVED** [review]

**Review by Marcellofabbri — APPROVED** [review]
LGTM

the new environment variable added to values.yaml may need to be added to cnp-flux-config too

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Pull request merged**